src/monodroid/jni/embedded-assemblies.cc

```diff
@@ -243,6 +243,35 @@ struct md_mmap_info {
 	size_t  size;
 };
 
+struct md_apk_mmap_info {
```

Comment:
Is this type actually necessary? `md_apk_mmap_info::mmap_info` appears to only  be used is within `md_mmap_apk_file()`, so couldn't `mmap_info` be function-local variables, as that information doesn't appear to be needed elsewhere?

Reply:
Sure that could be done, the only reason I kept this info was that at the start I was thinking about equivalent calls to munmap (otherwise, you'd need to realign the adress before munmap-ing, but we don't even do a call to munmap on those, and if we start doing it, it's just a realign so it's not really needed indeed.

Reply:
Fixed at fe1236c